support/scripts/mkukimg

```diff
@@ -73,6 +73,41 @@ EOF
 	fi
 }
 
+mkukmb2iso()
```

Comment:
All I have to say for this commit is to just name it `support/scripts: Support building multiboot2 images` as far as its commit subject goes. The fact that you did this minor rename for the old multiboot option is a small detail that can be mentioned in the main commit message only as you have already done.